test-plan.md

```diff
@@ -137,6 +140,9 @@ To setup a test project to verify on you can do:
 ##### C# Diagnostics
 * Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file. NOTE: This error squiggly will be misaligned due to known issues.
 
+#### TagHelper Quick Info
+* Typing `<environment></environment>` then hovering the mouse over the opening tag results in an informative window about that TagHelper.
```

Comment:
I'd provide an example with a disclaimer that the text may not be exact

Reply:
```suggestion
* Typing `<environment></environment>` then hovering the mouse over the opening tag results in a window about that TagHelper that should read something like `EnvironmentTagHelper ITagHelper implementation targeting <environment> elements that conditionally renders content based on the current value of IHostingEnvironment.EnvironmentName...`.
```